charts/postgres-operator/templates/operatorconfiguration.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/instance: {{ .Release.Name }}
 configuration:
-{{ toYaml .Values.configGeneral | indent 2 }}
+{{ tpl (toYaml .Values.configGeneral) . | indent 2 }}
```

Comment:
would it make sense to change it also for the other config sections?

Reply:
yes it would make sense

Reply:
could you do the same for `configLoadBalancer` and `configKubernetes` - I think it might only be required there

Reply:
From my point of view it's currently only needed in configKubernetes since this section contains an array. I'll create a separate pull request for that.